colortv/docs/androidTVSDK.md

```diff
@@ -792,19 +801,196 @@ Check our example [item layouts](https://github.com/color-tv/android-SampleApp/t
 
 This method is used to set the number of rows in RecyclerView with id `ctv_rvGrid` for a specified device type.
 
-#### setFont(Device device, Typeface typeface)
-
-This method is used to set a custom font for a specified device type. It only works if you don't use a custom item layout.
-
 #### setSnapEnabled(boolean enabled)
 
 This method is used to set snapping `RecyclerView` elements to the center of the screen. Available only on Mobile. We recommend to disable this option if the width of the elements is too small and the first or the last item is impossible to be snapped.
 
 - true - items snap to the center of the `RecyclerView`, video preview is enabled and will be shown if available.
 - false - list scrolls freely, video preview is disabled and will not be shown whether it is available or not.
 
+### UpNext customization
+
+It is possible to customize UpNext in 2 ways:
+
+- setting custom layout for UpNext
+- changing colors of default layout through public interface
+
+#### Using custom layout
+
+In order to set custom layout for UpNext you should do it appropriately to Recommendation Center using following method of `ColorTvContentRecommendationCongfig` instance:
+
+```java
+recommendationConfig.setUpNextLayout(Device device, @LayoutResource int layoutResId);
+```
+
+This method is used to set a custom UpNext layout for specified device type. You can use in that layout some specified views with proper ids which will be handled by sdk, and some recommendations data will be injected into them, but you can also add as many additional views as you want.
```

Comment:
`You can use some specified views with proper ids  in that layout, which will be handled by the sdk (some recommendations data will be injected into them), but`